config/config.go

```diff
@@ -90,130 +94,130 @@ type CurrencyPairFormatConfig struct {
 type Config struct {
 	Name              string
```

Comment:
These two settings (Name and EncryptConfig) can be made lowercase.

Reply:
good catch... missed these indeed